test/sql/test_automatic_partition/T/test_automatic_partition

```diff
@@ -166,3 +166,16 @@ insert into ss values('0001-02-01', 2);
 select * from ss;
 alter table ss drop partition p200201;
 select * from ss;
+
+-- name: test_two_replica
```

Comment:
The bug is occur sometimes. Could the test case trigger the bug 100 percent before the fixation?

Reply:
not 100 percent, but it can trigger sometimes, we have unstable test case check, if it fail we will know it.